cliquet/storage/memory.py

```diff
@@ -201,6 +201,7 @@ def create(self, collection_id, parent_id, record, id_generator=None,
         self.set_record_timestamp(collection_id, parent_id, record,
                                   modified_field=modified_field)
         self._store[collection_id][parent_id][_id] = record
+        self._cemetery[collection_id][parent_id].pop(_id, None)
```

Comment:
Actually ``pop`` is used there to remove it from the cemetery which is not handled by ``get``.